src/main/scala/sbtghactions/WorkflowStep.scala

```diff
@@ -36,21 +38,26 @@ object WorkflowStep {
     versions map {
       case jv @ JavaSpec(JavaSpec.Distribution.GraalVM(graalVersion), version) =>
         WorkflowStep.Use(
-          UseRef.Public("DeLaGuardo", "setup-graalvm", "5.0"),
+          UseRef.Public("graalvm", "setup-graalvm", "v1"),
           name = Some(s"Setup GraalVM (${jv.render})"),
           cond = Some(s"matrix.java == '${jv.render}'"),
-          params = Map(
-            "graalvm" -> graalVersion,
-            "java" -> s"java$version"))
+          params = ListMap(
+            "version" -> graalVersion,
+            "java-version" -> s"$version",
+            "components" -> "native-image",
+            // TODO: https://github.com/marketplace/actions/github-action-for-graalvm
+            // "github-token" -> s"$${{ secrets.GITHUB_TOKEN }}",
```

Comment:
@mdedetrich @armanbilge Looks like we already hit the API limit as we merged.

Reply:
https://github.com/sbt/sbt-github-actions/actions/runs/3906658205
So without the github-token, this action is not usable. Not sure if this requires additional action by the user.

Reply:
GitHub workflows automatically include a token.
https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
Here's follow up PR - https://github.com/sbt/sbt-github-actions/pull/141